src/js/select2/i18n/sl.js

```diff
@@ -0,0 +1,50 @@
+define(function () {
+  // English
+  return {
+    errorLoading: function () {
+      return 'Zadetkov iskanja ni bilo mogoče naložiti.';
+    },
+    inputTooLong: function (args) {
+      var overChars = args.input.length - args.maximum;
+
+      var message = 'Prosim zbrišite ' + overChars + ' znak';
+
+      if (overChars == 2)
+        message += 'a';
+      else if (overChars != 1) 
+        message += 'e';
+
+      return message;
+    },
+    inputTooShort: function (args) {
+      var remainingChars = args.minimum - args.input.length;
+
+      var message = 'Prosim vpišite še ' + remainingChars + ' znak';
+      if (remainingChars == 2)
```

Comment:
Within Select2, we enforce that braces are always used in conditionals. This largely has to do with having consistency across the code base.